src/Raven.Studio/typescript/viewmodels/resources/deleteResourceConfirm.ts

```diff
@@ -10,10 +10,12 @@ class deleteResourceConfirm extends confirmViewModelBase<deleteResourceConfirmRe
 
     keepFiles() {
         this.isKeepingFiles(true);
+        super.confirm();
```

Comment:
1. OK - using this.confirm()
2. It is not duplicate confirmation,
The 'Delete' btn was removed from html, so now there is a direct call to 'keepFile()' from the html, and a direct call to 'deleteEverything()' and I just call 'confirm()' to complete the work and close dialog.